Formula/o/opentofu.rb

```diff
@@ -0,0 +1,56 @@
+class Opentofu < Formula
+  desc "Drop-in replacement for Terraform. Infrastructure as Code Tool"
+  homepage "https://github.com/opentofu/opentofu"
+  url "https://github.com/opentofu/opentofu/archive/refs/tags/v1.6.0-alpha2.tar.gz"
+  sha256 "929105faff706c780acb89c2bbaa040a6110f9acba21e8b69751c848d64b521d"
+  license "MPL-2.0"
+  head "https://github.com/opentofu/opentofu.git", branch: "main"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+(?:[._-]alpha\d)?)$/i)
+  end
+
+  depends_on "go" => :build
+
+  # Needs libraries at runtime:
+  # /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by node)
+  fails_with gcc: "5"
```

Comment:
What is this about and is it still relevant? Is it shipping a secret `node` somewhere?

Reply:
Carried over from our terraform formula. Not sure if it's still relevant.

Reply:
Maybe it's because the prebuilt protobuf?

Reply:
would be good to try and remove it and see if it breaks

Reply:
(but can be after this PR is merged)